client/packages/host/src/components/Sync/Sync.tsx

```diff
@@ -111,6 +116,35 @@ export const Sync: React.FC = () => {
         </Row>
       </Grid>
       <SyncProgress syncStatus={syncStatus} isOperational={true} />
+      <Grid
+        container
+        flexDirection="column"
+        justifyContent="flex-start"
+        style={{ padding: '15 15 50 15', minWidth: 650 }}
+        marginTop={2}
+        flexWrap="nowrap"
+      >
+        <Typography variant="h5" color="primary" style={{ paddingBottom: 25 }}>
+          {t('heading.user-sync')}
+        </Typography>
+        <Row title={t('sync-info.last-successful-sync')}>
```

Comment:
I wonder if a default ( such as `never` ) would be possible? Maybe not required though - the user will be updated on login, so will never see this: right?

<img width="644" alt="Screenshot 2023-08-10 at 11 41 07 AM" src="https://github.com/openmsupply/open-msupply/assets/9192912/5ca54efc-7a6c-4837-ba81-18acabcca26e">

Reply:
I don't think it should ever show that since time is saved when the user first logs in as well... so there should always be a time